### DISCUSSION TITLE
Jekyll - Can't stop a deleted post from appearing

### DISCUSSION THREAD

**Discussion by lhornby** [community/community#27931] [Codespaces] [answered]
Hello,

I'm a bit of a novice at github pages and Jekyll. I am setting up a [website](https://lhornby.github.io/) using the al-folio theme 

The issue I have is that I cannot delete an entry on the blog which was an example post in the repo. Is the entry that reads "Displaying External Posts on Your al-folio Blog".  

I have deleted it from '_posts' and looking through the actions, I can't see any failures that would prevent it from being actually deleted. Am I missing something? Is it stuck in a different branch? I believe the file would have been called "2022-23-04-redirect.md".

**Comment by radar-blip** [comment]
I can see what you're talking about on the page here: https://lhornby.github.io/blog/
But I cant find the page/code that displays "Displaying External Posts..."
I'm thinking you're right about it being stuck in a different branch.

Also, some of your branches are *ahead of* and *behind* others, It might just be a matter of organizing/updating.

**Reply by lhornby** [reply]
I would love to figure out the root cause but I think for now I'm going to scrap the fork and start again.

**Reply by radar-blip** [reply]
That might be your best bet. You'll learn where things went wrong along the way. 
Good Luck! 👍🏼

**Comment by lhornby** [comment, marked as answer]
Solution:

in config.yml comment out these lines:

```
# External sources.
# If you have blog posts published on medium.com or other exteranl sources,
# you can display them in your blog by adding a link to the RSS feed.
external_sources:
 - name: medium.com
 -  rss_url: https://medium.com/@al-folio/feed
```
